profiles/sun0225SUN.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "小孙同学",
+    "githubUsername": "sun0225SUN",
+    "imageUrl": "https://user-images.githubusercontent.com/79169717/151289290-2900e1a6-af16-4f72-9341-444d6d71fbc3.png",
```

Comment:
I recommend more of an image like this. What do you think? 😃

![image](https://user-images.githubusercontent.com/51391473/151290637-01ac43a5-aa07-4bf4-853c-7031edd69b4b.png)

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/151290637-01ac43a5-aa07-4bf4-853c-7031edd69b4b.png",
```

Reply:
ok , i think so. 😘
Because the size of this image is only 158kb.😅
![](https://user-images.githubusercontent.com/51391473/151290637-01ac43a5-aa07-4bf4-853c-7031edd69b4b.png)
However, I look at this image a bit unpretty, I will compress it later and re-upload it.❤️

Reply:
> I recommend more of an image like this. What do you think? 😃
> 
> ![image](https://user-images.githubusercontent.com/51391473/151290637-01ac43a5-aa07-4bf4-853c-7031edd69b4b.png)

I changed the picture and I think it works now, thank you so much!🥰